### PR TITLE
Update README with status and access definitions

### PR DESCRIPTION
**Pull request by velsietis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,18 +86,25 @@ Name of the team that is the main point of contact for information about this AP
...
 
-The status of an API is determined by availability.
+Status indicating the level of stability and support, defined as follows:
 
-- **Live**: in Production.
-- **Beta**: in Production but for pilot use only.
-- **Alpha**: in SIT but not in production.
-- **Experimental**: in internal Dev environment or Sandbox but not in SIT or production.
+- `experimental`: Proof of Concept or Prototype still under development.
+- `alpha`: In a test (SIT) environment and undergoing formal evaluation by a tester
```
Comment (Geoff1900) [inline README.md]: I don't see any added value in adding the abbreviation _SIT_, particularly as it's not explained or used elsewhere.

**Review by Geoff1900 — APPROVED** [review]
Approved but not quite sure whether there's value in expanding abbreviations where they may not be commonly understood - e.g. _PSBA_ and _HSCN_

File: README.md (hunk 1 above)
Comment (velsietis) [reply README.md]: Yeah, I was unsure about whether it's needed, deleted and added a few times, so happy to delete it again.  Although as an abbreviation I think it's fine as it's a commonly used term in software development, not specific to NWIS (maybe I should have called it WSIT, now there a thought...)

```suggestion
- `alpha`: In a test environment and undergoing formal evaluation by a tester
```

**Comment by velsietis** [comment]
> Approved but not quite sure whether there's value in expanding abbreviations where they may not be commonly understood - e.g. _PSBA_ and _HSCN_

I did consider explaining them, but it's not so relevant and those who know will know, those who don't can easily google it

**Pull request merged**